boards/opentitan/src/main.rs

```diff
@@ -482,6 +493,49 @@ unsafe fn setup() -> (
     sip_hash.set_client(tickv);
     TICKV = Some(tickv);
 
+    let mux_kv = components::kv_system::KVStoreMuxComponent::new(tickv).finalize(
+        components::kv_store_mux_component_helper!(
+            capsules::tickv::TicKVStore<
+                capsules::virtual_flash::FlashUser<lowrisc::flash_ctrl::FlashCtrl>,
+                capsules::sip_hash::SipHasher24<'static>,
+            >,
+            capsules::tickv::TicKVKeyType,
+        ),
+    );
+
+    let kv_store_key_buf = static_init!(capsules::tickv::TicKVKeyType, [0; 8]);
+    let header_buf = static_init!([u8; 9], [0; 9]);
+
+    let kv_store =
+        components::kv_system::KVStoreComponent::new(mux_kv, kv_store_key_buf, header_buf)
+            .finalize(components::kv_store_component_helper!(
+                capsules::tickv::TicKVStore<
+                    capsules::virtual_flash::FlashUser<lowrisc::flash_ctrl::FlashCtrl>,
+                    capsules::sip_hash::SipHasher24<'static>,
+                >,
+                capsules::tickv::TicKVKeyType,
+            ));
+    tickv.set_client(kv_store);
```

Comment:
It feels like this client setting belongs in the components if possible

Reply:
We currently don't pass `tickv` to the component as we don't need it. I have left this here, but let me know if you feel strongly and I can move it into the component.